helm/private/runner/BUILD.bazel

```diff
@@ -0,0 +1,40 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary")
+
+package(default_visibility = ["//visibility:public"])
+
+exports_files([
+    "runner_wrapper.bat.template",
+    "runner_wrapper.sh.template",
+])
+
+config_setting(
+    name = "windows",
```

Comment:
I think this is actually unnecessary in modern versions of Bazel and you can use `@platforms//os:windows` directly

Reply:
Updated all of these to use `@platforms//os:windows`.